workinggroups/networking.md

```diff
@@ -0,0 +1,53 @@
+# CNCF Networking WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore cloud native networking technology and concepts around the container networking interface (CNI).
+
+## Goals and Expected Outcomes
+
+* Recommend CNI be adopted as initial network interface specification focused on connectivity and portability as an official CNCF project.
```

Comment:
I assume this is done. Otherwise, it should be under the CNI project rather than in a WG.